.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm run validate:precommit
+npx lint-staged
```

Comment:
Real small thing, could we change this to something like?

```suggestion
npm run lint-staged
```

And add that script to `package.json`? There's just a little more overhead for running `npx`, and we already have `lint-staged` added to the `devDependencies`